src/components/RetipComponent.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <div v-if="toggleForm" class="clearfix retip__container">
+    <div class="input-group mr-1 float-left">
+      <input type="number" step="0.000001" v-model="value" @input="onValueChange" class="form-control" aria-label="Default">
+      <div class="input-group-append">
+        <span class="input-group-text append__ae"> <span class="ae">AE</span>&nbsp;(~ {{current.currency.toUpperCase()}} {{ fiatValue }} )</span>
```

Comment:
You can use `fiat-value` component here for calculating and displaying the fiat value.